libnavigation-core/src/main/java/com/mapbox/navigation/core/formatter/MapboxDistanceUtil.kt

```diff
@@ -234,4 +240,38 @@ object MapboxDistanceUtil {
         }
         return this.createConfigurationContext(config).resources
     }
+
+    private enum class FormattingRange { INVALID, SMALL, MEDIUM, LARGE }
+
+    private fun getFormattingRange(
+        distanceInMeters: Double,
+        unitType: UnitType
+    ): FormattingRange {
+        if (distanceInMeters !in 0.0..Double.MAX_VALUE) {
+            return FormattingRange.INVALID
```

Comment:
could you also add tests for the `INVALID` format (`distance` is negative)

Reply:
Sure, always happy to add tests. :)